client/src/components/overview/Cart.jsx

```diff
@@ -1,46 +1,77 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 
 const Cart = function ({ skus }) {
   const [size, setSize] = useState(null);
 
   const skusArray = [];
   const hash = {};
+  const selectSize = 'Select Size';
+
+  useEffect(() => {
+    setSize(null)
+  }, [skus])
+
 
   for (const key in skus) {
-    skusArray.push({ key: skus[key] });
+    if (key == 2275494) {
+      skusArray.push({ key: {quantity: skus[key].quantity, size: 'XXL'}})
```

Comment:
How come this is hardcoded? If it's an edge case, what if there are other SKUs with the same edge case?

Reply:
If I'm correct, all SKU ID's are unique and this was the only product with XXL represented as XL. All other products had XL and XXL while this specific style had two XL.

Reply:
Gotcha, I think that's a reasonable assumption (and would ideally be fixed by telling the API provider). Looks good!